frontend/controller/actions/identity.js

```diff
@@ -359,41 +283,13 @@ export default (sbp('sbp/selectors/register', {
         return index === -1 ? contractSyncPriorityList.length : index
       }
 
-      // loading the website instead of stalling out.
-      try {
-        if (!state) {
-          // Make sure we don't unsubscribe from our own identity contract
-          // Note that this should be done _after_ calling
-          // `chelonia/storeSecretKeys`: If the following line results in
-          // syncing the identity contract and fetching events, the secret keys
-          // for processing them will not be available otherwise.
-          await sbp('chelonia/contract/retain', identityContractID)
-        } else {
-          // If there is a state, we've already retained the identity contract
-          // but might need to fetch the latest events
-          await sbp('chelonia/contract/sync', identityContractID, { force: true })
-        }
-      } catch (err) {
-        sbp('okTurtles.events/emit', LOGIN_ERROR, { username, identityContractID, error: err })
-        const errMessage = err?.message || String(err)
-        console.error('Error during login contract sync', errMessage)
-
-        const promptOptions = {
-          heading: L('Login error'),
-          question: L('Do you want to log out? {br_}Error details: {err}.', { err: err.message, ...LTags() }),
-          primaryButton: L('No'),
-          secondaryButton: L('Yes')
-        }
-
-        const result = await sbp('gi.ui/prompt', promptOptions)
-        if (!result) {
-          return sbp('gi.actions/identity/logout')
-        } else {
-          throw err
-        }
-      }
-
       await sbp('gi.actions/identity/kv/load')
+      // NOTE: update chatRoomUnreadMessages to the latest one we do this here
+      //       just after the identity contract is synced because
+      //       while syncing the chatroom contract it could be necessary to update chatRoomUnreadMessages
+      await sbp('gi.actions/identity/kv/loadChatRoomUnreadMessages')
+      // NOTE: load users preferences config which is saved in KV store
+      await sbp('gi.actions/identity/kv/loadPreferences')
```

Comment:
These selectors are already called by line 286 above so should be removed here

Reply:
That's right.